static/app/types/system.tsx

```diff
@@ -207,6 +207,7 @@ export interface Config {
   relocationConfig?: {
     selectableRegions: string[];
   };
+  shouldShowBeaconConsentPrompt?: boolean;
```

Comment:
nit: Isn't something like `resourceUsageConsentPrompt` a bit clearer? At first I thought this referred to the entire beacon prompt, not just the resource usage question.

Reply:
I specifically used `shouldShowBeaconConsentPrompt` because the web page that shows up can be reused in the future